src/rdkafka.h

```diff
@@ -6631,6 +6794,38 @@ rd_kafka_AdminOptions_set_broker(rd_kafka_AdminOptions_t *options,
                                  size_t errstr_size);
 
 
+/**
+ * @brief Whether broker should return stable offsets
+ *        (transaction-committed).
+ *
+ * @param options Admin options.
+ * @param true_or_false Defaults to false.
+ *
+ * @return NULL on success, a new error instance that must be
+ *         released with rd_kafka_error_destroy() in case of error.
+ *
+ * @remark This option is valid for ListConsumerGroupOffsets.
+ */
+RD_EXPORT
+rd_kafka_error_t *rd_kafka_AdminOptions_set_require_stable_offsets(
```

Comment:
More than a review comment, a question - Why are we changing the return type of admin option from rd_kafka_resp_err_t coupled with two out params, to a single rd_kafka_error_t?

Is this the new way going forward?

Reply:
Looking at it a bit:
Is this comment the norm for all API including AdminOptions_set* going forward? 
https://github.com/edenhill/librdkafka/pull/3995#discussion_r1029752282

Reply:
Yes, we want to return only error objects, so the user doesn't check the error code but relies on flags related to actions like retriable, abortable, fatal and so on.